src/status_im/protocol/web3/keys.cljs

```diff
@@ -13,14 +14,20 @@
 
 (defn get-sym-key
   "Memoizes expensive calls by password."
-  [web3 password callback]
-  (if-let [key-id (get @password->keys password)]
-    (callback key-id)
-    (add-sym-key-from-password
+  ([web3 password success-fn]
```

Comment:
Nice growable software by extending capability and not removing it :)

Reply:
i think changing usages of `get-sym-key` is outside of scope of this PR. do you think it should be done right away?

Reply:
Oh not at all, it was just a compliment!